modules/smartadserverBidAdapter.js

```diff
@@ -332,18 +331,22 @@ export const spec = {
    * @param {string} mediaType Bid media type
    * @return {number} Floor price
    */
-  getBidFloor: function (bid, currency, mediaType) {
-    if (!isFn(bid.getFloor)) {
-      return DEFAULT_FLOOR;
+  getBidFloor: (bid, currency, mediaType) => {
+    const floors = [];
+
+    if (isFn(bid.getFloor)) {
+      (deepAccess(bid, `mediaTypes.${mediaType}.${mediaType === BANNER ? 'sizes' : 'playerSize'}`) || []).forEach(size => {
+        const floor = bid.getFloor({
+          currency: currency || 'USD',
+          mediaType,
+          size
+        }).floor;
+
+        floors.push(!isNaN(floor) ? floor : DEFAULT_FLOOR);
+      });
     }
 
-    const floor = bid.getFloor({
-      currency: currency || 'USD',
-      mediaType,
-      size: '*'
-    });
-
-    return isPlainObject(floor) && !isNaN(floor.floor) ? floor.floor : DEFAULT_FLOOR;
+    return floors.length ? Math.min(...floors) : DEFAULT_FLOOR;
```

Comment:
line 357 below is throwing a warning:

/home/circleci/Prebid.js/modules/smartadserverBidAdapter.js
  357:1  warning  The type 'syncs' is undefined  jsdoc/no-undefined-types

Reply:
Fixed.